Source/Manager/UserTokensStorage.swift

```diff
@@ -66,4 +66,19 @@ struct UserTokensStorage {
             log(level: .error, from: self, "error removing keychain for user: \(error)", force: true)
         }
     }
+
+    func clearAll() throws {
+        let keychain = UserTokensKeychain()
+        keychain.removeAllTokens()
+
+        do {
+            try keychain.saveInKeychain()
```

Comment:
To next time. What is the difference between UserTokensKeychain and SPiDKeychainWrapper?

Reply:
SPiDKeychainWrapper reimplements the keychain how the old SPID SDK used to do it (copy pasted from there) and UserTokensKeychain is the current keychain.